build.sbt

```diff
@@ -26,6 +27,15 @@ val dep = {
     "ch.qos.logback" % "logback-classic" % "1.1.9",
     "org.scalatest" %% "scalatest" % "3.0.1" % "it,test",
     "org.scalacheck" %% "scalacheck" % "1.13.4" % "it,test",
+    "org.scalacheck" %% "scalacheck" % "1.13.4" % "it,test",
+    "org.scorexfoundation" %% "iodb" % "0.2.0",
+    "ch.qos.logback" % "logback-classic" % "1.1.9",
+    "org.jline" % "jline" % "3.1.2",
+    "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.5",
+    "io.circe" %% "circe-core" % circeVersion,
```

Comment:
We're all agreed on a single dep. I thought this came up before (in Poland) and we decided on akka-http-spray-json? Circe is new and popular. I'll bring this up at VSU.